# Protocol

## DNA extraction

### Checklist

-   Extraction kit - [DNeasy Powersoil Pro, Qiagen, Hilden, Germany](https://www.qiagen.com/au/resources/resourcedetail?id=9bb59b74-e493-4aeb-b6c1-f660852e8d97&lang=en)
-   Alternatively the high throughput version of the kit used with the QIAcube [DNeasy 96 PowerSoil Pro QIAcube HT Kit](https://rmiteduau-my.sharepoint.com/personal/christian_krohn_rmit_edu_au/Documents/Experiments/2025_ETPSequencingProject/_%09https:/www.qiagen.com/au/products/discovery-and-translational-research/dna-rna-purification/dna-purification/microbial-dna/dneasy-96-powersoil-pro-qiacube-ht-kit)
-   Vortex with 24 x 1.5 mL tube adapter (e.g. Vortex Genie 2 + adapter). Alternatively, a PowerLyzer Homogenizer.
-   NanoDrop spectrophotometer to assess DNA quality.
-   Qubit fluorometer for accurately measure DNA concentrations.

### Process

-   Follow the extraction kit's protocol with 15 mins bead beating with Genie vortex and 24-tube adapter. Reduce this to 10 mins if less than 24 samples are extracted or if shearing of DNA should be minimised.
-   Measure DNA quality using DNA extract (1 µL) using a Nanodrop spectrophotometer
-   Measure DNA concentration using a Qubit fluorometer.
-   Into a 96-well plate, normalise extracted DNA to required PCR concentrations. For example, if 10 ng template is required for PCR, normalise DNA to 5 ng/µL and use 2 µL.\
-   Store DNA at 4˚C until library preparation - no more than 1 week.
-   Store DNA at -20˚C if sequencing is more than 1 week away.

## DNA library preparation and PCR

### DNA Library preparation including PCR, multiplexing and pooling of samples.

### Checklist

-   10 mM dNTPs (N0447S, New England Biolab)
-   Q5 Hot Start High-Fidelity DNA Polymerase (M0493, New England Biolab)
-   10 µM Forward Primer A519F (CAGCMGCCGCGGTAA) [@Martijn2019]
-   10 µM Reverse Primer U2428R (CCRAMCTGTCTCACGACG) [@Martijn2019]
-   JetSeq Clean Magnetic beads - or equivalent (MER-BIO-68031, Millenium Science) - to clean up PCR products\
-   twin.tec® PCR plate 96 LoBind, semi skirted (0030129504, Eppendorf)\
-   Nuclease-free water
-   10 mM Tris-HCl pH 8.0 with 50 mM NaCl (UltraPure™ 1M Tris-HCI, pH 8.0 #15568025 & NaCl (5 M), RNase-free #AM9760G, Thermo Fisher)\
-   HyperLadder 1kb (BIO-33025, Bioline, Millenium Science)
-   80% ethanol, freshly prepared in nuclease-free water
-   Qubit 1X dsDNA BR Assay Kit (Q33266, Thermo Fisher)

**Notes**

-   Do not vortex tubes during library preparation and use wide-bore tips to prevent DNA fragmentation. Fragmentation of amplicons may lead to incomplete reads.\
-   The primer amplifies the whole rrn operon.

Benefits of targeting the whole rrn operon:

-   Superior species-level resolution and accuracy [@Cusco2018; @Srinivas2024].\
-   Covers Bacteria and Archaea.

Risks of targeting the whole rrn operon:

-   Not as representative of true abundances as full-length 16S amplicons.\
-   Species with unlinked 16S and 23S rrn DNA will be missed with this approach (for example \< 9% of rRNA genes in wastewater sludge [@Brewer2020]).

### PCR

Time required \~4 hrs incl. 2hrs, 40mins PCR.

-   Prepare a mastermix for required number of 50 µl reactions.
-   Add 3 µl of eDNA (5ng /µl) into a 96-well plate (e.g. Eppendorf twin.tec® PCR plate 96 LoBind, semi-skirted) using a multichannel pipette.\
-   Add 47 µl of mastermix using a multichannel pipette and carefully pipette up and down 10 x
-   Run thermocycler.
-   Verify amplification length via 1% agarose gel electrophoresis \@ 100V for 30 min including a 1 kb ladder (Fig. 2).
-   Store at 4˚C overnight if needed.

**Important**:

-   Use hot start polymerase for ease of use, with high fidelity/accuracy and one that is suitable for long amplicons. For example, Q5 High- Fidelity DNA Polymerase kit (New England Biolabs) with GC enhancer [@Martijn2019].\
-   ≤ 200 fmol is required per sample for the Native Barcoding Kit from ONP. Based on 4.25 kb (4-4.5kb), the final DNA concentration after cleaning up PCR products should be no less than 48 ng/µL (at 11.5 µl input volume) - giving 552 ng of 4.25kb amplicons.\
-   It may require two–three PCR reactions to achieve the required DNA amount (200 fmol); e.g. pool 2–3 x 50µl PCR products, clean combined and elude in 32.5 µl Tris.
  
  
```{r table, tidy=FALSE, echo=FALSE }
library(tidyverse)
pcr <- read.csv("PCR.csv",header=TRUE, stringsAsFactors=FALSE, fileEncoding="latin1")
pcr <- pcr %>% dplyr::rename(`50 µl reaction` = "Fifty.microL.Reaction" )
pcr <- pcr %>% dplyr::rename(`Final concentration` = "Final.Concentration" )
knitr::kable(
 pcr, caption = 'List of components for each reaction (each tube) for PCR. See https://www.neb.com/en-au/protocols/2012/08/30/pcr-using-q5-hot-start-high-fidelity-dna-polymerase-m0493 for details.  ',
 booktabs = TRUE
)
```
