# Protocol

## DNA extraction

### Checklist

-   Extraction kit - [DNeasy Powersoil Pro, Qiagen, Hilden, Germany](https://www.qiagen.com/au/resources/resourcedetail?id=9bb59b74-e493-4aeb-b6c1-f660852e8d97&lang=en). See Section \@ref(prerequisites) for more details.
-   Alternatively the high throughput version of the kit used with the QIAcube [DNeasy 96 PowerSoil Pro QIAcube HT Kit](https://rmiteduau-my.sharepoint.com/personal/christian_krohn_rmit_edu_au/Documents/Experiments/2025_ETPSequencingProject/_%09https:/www.qiagen.com/au/products/discovery-and-translational-research/dna-rna-purification/dna-purification/microbial-dna/dneasy-96-powersoil-pro-qiacube-ht-kit).
-   Vortex with 24 x 1.5 mL tube adapter (e.g. Vortex Genie 2 + adapter). Alternatively, a PowerLyzer Homogenizer.
-   NanoDrop spectrophotometer to assess DNA quality.
-   Qubit fluorometer to accurately measure DNA concentrations.

### Steps

-   Follow the extraction kit's protocol with 15 mins bead beating using a Vortex-Genie and 24-tube adapter. Reduce this to 10 mins if less than 24 samples are extracted or if shearing of DNA should be minimised.
-   Measure DNA quality using DNA extract (1 µL) using a Nanodrop spectrophotometer
-   Measure DNA concentration using a Qubit fluorometer.
-   Into a 96-well plate, normalise extracted DNA to required PCR concentrations. For example, if 10 ng template is required for PCR, normalise DNA to 5 ng/µL.\
-   Store DNA at 4˚C until library preparation - no more than 1 week.
-   Store DNA at -20˚C if sequencing is more than 1 week away.

## Amplification of 16S-ITS-23 operon

### Checklist

-   10 mM dNTPs (N0447S, New England Biolab)
-   Q5 Hot Start High-Fidelity DNA Polymerase (M0493, New England Biolab)
-   10 µM Forward Primer A519F (CAGCMGCCGCGGTAA) [@Martijn2019]
-   10 µM Reverse Primer U2428R (CCRAMCTGTCTCACGACG) [@Martijn2019]
-   JetSeq Clean Magnetic beads - or equivalent (MER-BIO-68031, Millenium Science) - to clean up PCR products\
-   twin.tec® PCR plate 96 LoBind, semi skirted (0030129504, Eppendorf)\
-   Nuclease-free water
-   10 mM Tris-HCl pH 8.0 with 50 mM NaCl (UltraPure™ 1M Tris-HCI, pH 8.0 #15568025 & NaCl (5 M), RNase-free #AM9760G, Thermo Fisher)\
-   HyperLadder 1kb (BIO-33025, Bioline, Millenium Science)
-   80% ethanol, freshly prepared in nuclease-free water
-   Qubit 1X dsDNA BR Assay Kit (Q33266, Thermo Fisher)

**Notes**

-   Do not vortex tubes during library preparation and use wide-bore tips to prevent DNA fragmentation. Fragmentation of amplicons may lead to incomplete reads.\
-   The primer amplifies the whole rrn operon.

Benefits of targeting the whole rrn operon:

-   Superior species-level resolution and accuracy [@Cusco2018; @Srinivas2024].\
-   Covers Bacteria and Archaea.

Risks of targeting the whole rrn operon:

-   Not as representative of true abundances as full-length 16S amplicons.\
-   Species with unlinked 16S and 23S rrn DNA will be missed with this approach (for example \< 9% of rRNA genes in wastewater sludge [@Brewer2020]).

### PCR

Time required \~4 hrs incl. 2hrs, 40mins PCR.

-   Prepare a mastermix for required number of 50 µl reactions (Table 3.1). Note that in our experience at least two 50 µl reactions are required to produce sufficient amplicon concentrations (200 fmol) for the Native Barcoding Kit from Oxford Nanopore.\
-   Add 3 µl of eDNA (5ng /µl) into a 96-well plate (e.g. Eppendorf twin.tec® PCR plate 96 LoBind, semi-skirted) using a multichannel pipette.\
-   Add 47 µl of mastermix using a multichannel pipette and carefully pipette up and down 10 x
-   Run thermocycler (Table 3.2).
-   Verify amplification length via 1% agarose gel electrophoresis \@ 100V for 30 min including a 1 kb ladder.
-   Store at 4˚C overnight if needed.

**Important**:

-   Use hot start polymerase for ease of use, with high fidelity/accuracy and one that is suitable for long amplicons. For example, Q5 High- Fidelity DNA Polymerase kit (New England Biolabs) with GC enhancer [@Martijn2019].\
-   ≤ 200 fmol is required per sample for the Native Barcoding Kit from ONP. Based on 4.25 kb (4-4.5kb), the final DNA concentration after cleaning up PCR products should be no less than 48 ng/µL (at 11.5 µl input volume) - giving 552 ng of 4.25kb amplicons.\
-   It may require two–three PCR reactions to achieve the required DNA amount (200 fmol); e.g. pool 2–3 x 50µl PCR products, clean combined and elude in 32.5 µl Tris.

   

```{r table, tidy=FALSE, echo=FALSE, message=FALSE, tidy=FALSE}
library(tidyverse)
pcr <- read.csv("PCR.csv",header=TRUE, stringsAsFactors=FALSE, fileEncoding="latin1")
pcr <- pcr %>% dplyr::rename(`50 µl reaction` = "Fifty.microL.Reaction" )
pcr <- pcr %>% dplyr::rename(`Final concentration` = "Final.Concentration" )
knitr::kable(
 pcr, caption = 'List of components for each reaction (each tube) for PCR. See https://www.neb.com/en-au/protocols/2012/08/30/pcr-using-q5-hot-start-high-fidelity-dna-polymerase-m0493 for details.  ',
 booktabs = TRUE
)
```

 

```{r table2, tidy=FALSE, echo=FALSE, message=FALSE, tidy=FALSE}
library(tidyverse)
pcrcond <- read.csv("PCRcond.csv",header=TRUE, stringsAsFactors=FALSE, fileEncoding="latin1")
pcrcond <- pcrcond %>% dplyr::rename(`Cycle conditions` = "Cycle.conditions" )
knitr::kable(
 pcrcond, caption = 'Thermocycler conditions (Martijn et al 2019).',
 booktabs = TRUE
)
```

\

## PCR product clean-up

Clean with 0.6X JetSeq Clean Magnetic beads (or equivalent) and wash twice with 80% ethanol. This follows a similar protocol to the Illumina 16S-metagenomics library prep guide.

Time required \~ 1 hr per 24 samples.

### Checklist

-   Magnet for 96-well plate
-   80% ethanol, freshly prepared in nuclease-free water\
-   Nuclease-free water
-   10 mM Tris-HCl pH 8.0 with 50 mM NaCl (UltraPure™ 1M Tris-HCI, pH 8.0 #15568025 & NaCl (5 M), RNase-free #AM9760G, Thermo Fisher)
-   JetSeq Clean Magnetic beads (MER-BIO-68031, Millenium Science) or equivalent - to clean up PCR products.

**Notes**

Use wide bore tips for adding mag beads and subsequent (careful) mixing by pipetting to minimise DNA fragmentation.

### Steps

-   Pool replicate reactions into one well - e.g. 3 x 50 µl = 150 µl in one well.
-   Using a multi channel pipette, add 0.6X beads to PCR products in the same wells (e.g. 90 µl to 150 µl pooled reactions), pipette 10 x up and down and/or a gentle vortex after sealing the plate, careful not to spill (Vortex Genie speed 2).
-   10 min incubation at RT
-   Place 5 mins on 96-well magnet and remove supernatant.
-   Wash twice with 200 µl 80% OH in same 96-well plate and let alcohol evaporate for \~ 5 mins or until alcohol is completely removed.
-   Take plate off magnet and elude pellet to a volume that provides appropriate DNA concentrations
    -   e.g. add 32.5 µl of elution buffer (10 mM Tris 50mM NaCl). The pellet will be stuck higher than the buffer surface in the well.
    -   Hence, pipetting up and down 20–30 x may be necessary to carefully wash the pellet of the tube wall, using a wide bore tip.
    -   Or centrifuge plate at 200-500 rcf for 1 min.
-   Off-magnet, incubate in final elution (incl. beads) for 15 minutes at 37°C. Every 2 minutes, agitate the sample by gently vortexing the plate for 10 seconds (after sealing plate) to encourage DNA elution.
-   Collect 30 µl of cleaned DNA after \~ 5 mins on magnet using wide bore tips.
-   Quantify and record amplicon concentrations using Qubit HS or BS chemistry and record values. Based on 4.25 kb amplicons (4-4.5kb), the final DNA concentration after cleaning up PCR products should be no less than 48 ng/µL to have sufficient material for downstream library preparation (200 fmol, \> 552 ng of 4.25kb amplicons).
-   Verify amplification length via 1% agarose gel electrophoresis \@ 100V for 30 min including a 1 kb ladder.
-   For each sample calculate volumes required of cleaned DNA and H~2~0 to get 200 fmol in 11.5 µl - for library preparation with the Native Barcoding Kit from Oxford Nanopore. Use V1 = C2\*V2 / C1, where V1 = x, C1 = DNA concentrations (ng/µl), C2 = 48 ng/µl, V2 = 11.5 µl). H~2~0 volume = 11.5 µl - V1. These volumes will be used in the next step.

## Library preparation

DNA Library preparation, including multiplexing and pooling of different samples. Here the 16S-ITS-23S rRNA operon was amplified from environmental DNA during PCR.

**Notes**

-   Follow the protocol **Native Barcoding Kit 96 V14 (SQK-NBD114.96)** from Oxford Nanopore.
-   Update the MinION packages and MinKNOW software.
-   Prepare a sample sheet for upload to instrument (to save time when setting up the sequencing in MinKNOW)
-   Check and record which basecaller is used in the most recent MINKNOW software update. Should be Dorado.
-   Print out and follow the Kit's protocol and checklist (`Ligation sequencing amplicons - Native Barcoding Kit 96 V14 (SQK-NBD114.96)-minion.pdf`).
-   Amplicon product yields from PCR can vary widely depending on environmental source.
    -   To achieve consistent (normalised) concentrations (200 fmol) across all samples, add appropriate input volume (calculated with C1*V1 = C2*V2) and fill to 11.5 µl with H20 to the first step of the Native Barcoding kit protocol ('Ligation sequencing amplicons - Native Barcoding Kit 96 V14 (SQK-NBD114.96)-minion.pdf'.
    -   If concentration was lower than the required 200 fmol for a sample, the input volume of the end-prepped amplicons into the second step of the protocol (native barcode ligation step) may have to be increased accordingly.

### Outline of steps

-   End-prep DNA of 11.5 µL DNA (200 fmol).
-   Native barcode ligation using 0.75 µL of end-prepped DNA using one of 96 barcodes from the kit incl. pooling of barcoded samples into LoBind tube (1.5 mL).
-   Bead clean-up and elution of 35 µL of pooled library and Qubit.
-   SAFE STOP: store at 4 ˚C overnight if needed.
-   Adapter ligation to get a final library of 35–50 fmol at 12 µL.

