<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</title>
  <meta name="description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  <meta name="github-repo" content="ABlab-workflows-longreads" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]" />
  
  <meta name="twitter:description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  

<meta name="author" content="Christian Krohn, PhD, RMIT University" />


<meta name="date" content="2024-06-22" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="prerequisites.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Andy Ball lab RMIT</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> About this GitBook</a></li>
<li class="chapter" data-level="2" data-path="prerequisites.html"><a href="prerequisites.html"><i class="fa fa-check"></i><b>2</b> Prerequisites, equipment and consumables</a>
<ul>
<li class="chapter" data-level="2.1" data-path="prerequisites.html"><a href="prerequisites.html#prerequisites-1"><i class="fa fa-check"></i><b>2.1</b> Prerequisites</a></li>
<li class="chapter" data-level="2.2" data-path="prerequisites.html"><a href="prerequisites.html#dna-extraction-kit"><i class="fa fa-check"></i><b>2.2</b> DNA extraction kit</a></li>
<li class="chapter" data-level="2.3" data-path="prerequisites.html"><a href="prerequisites.html#other-consumables"><i class="fa fa-check"></i><b>2.3</b> Other consumables</a></li>
<li class="chapter" data-level="2.4" data-path="prerequisites.html"><a href="prerequisites.html#protocols"><i class="fa fa-check"></i><b>2.4</b> Protocols</a></li>
<li class="chapter" data-level="2.5" data-path="prerequisites.html"><a href="prerequisites.html#equipment"><i class="fa fa-check"></i><b>2.5</b> Equipment</a></li>
<li class="chapter" data-level="2.6" data-path="prerequisites.html"><a href="prerequisites.html#computational-resources-and-database"><i class="fa fa-check"></i><b>2.6</b> Computational resources and database</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="protocol.html"><a href="protocol.html"><i class="fa fa-check"></i><b>3</b> Protocol</a>
<ul>
<li class="chapter" data-level="3.1" data-path="protocol.html"><a href="protocol.html#dna-extraction"><i class="fa fa-check"></i><b>3.1</b> DNA extraction</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="protocol.html"><a href="protocol.html#checklist"><i class="fa fa-check"></i><b>3.1.1</b> Checklist</a></li>
<li class="chapter" data-level="3.1.2" data-path="protocol.html"><a href="protocol.html#steps"><i class="fa fa-check"></i><b>3.1.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="protocol.html"><a href="protocol.html#amplification-of-16s-its-23-operon"><i class="fa fa-check"></i><b>3.2</b> Amplification of 16S-ITS-23 operon</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="protocol.html"><a href="protocol.html#checklist-1"><i class="fa fa-check"></i><b>3.2.1</b> Checklist</a></li>
<li class="chapter" data-level="3.2.2" data-path="protocol.html"><a href="protocol.html#pcr"><i class="fa fa-check"></i><b>3.2.2</b> PCR</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="protocol.html"><a href="protocol.html#pcr-product-clean-up"><i class="fa fa-check"></i><b>3.3</b> PCR product clean-up</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="protocol.html"><a href="protocol.html#checklist-2"><i class="fa fa-check"></i><b>3.3.1</b> Checklist</a></li>
<li class="chapter" data-level="3.3.2" data-path="protocol.html"><a href="protocol.html#steps-1"><i class="fa fa-check"></i><b>3.3.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="protocol.html"><a href="protocol.html#library-preparation"><i class="fa fa-check"></i><b>3.4</b> Library preparation</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="protocol.html"><a href="protocol.html#checklist-3"><i class="fa fa-check"></i><b>3.4.1</b> Checklist</a></li>
<li class="chapter" data-level="3.4.2" data-path="protocol.html"><a href="protocol.html#steps-2"><i class="fa fa-check"></i><b>3.4.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="protocol.html"><a href="protocol.html#priming-and-loading-of-flow-cell---run-start"><i class="fa fa-check"></i><b>3.5</b> Priming and loading of flow cell - run start</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="protocol.html"><a href="protocol.html#checklist-4"><i class="fa fa-check"></i><b>3.5.1</b> Checklist</a></li>
<li class="chapter" data-level="3.5.2" data-path="protocol.html"><a href="protocol.html#steps-3"><i class="fa fa-check"></i><b>3.5.2</b> Steps</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</h1>
<p class="author"><em><a href="https://www.rmit.edu.au/contact/staff-contacts/academic-staff/k/krohn---christian">Christian Krohn, PhD, RMIT University</a></em></p>
<p class="date"><em>2024-06-22</em></p>
</div>
<div id="about" class="section level1 hasAnchor" number="1">
<h1><span class="header-section-number">Chapter 1</span> About this GitBook<a href="index.html#about" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p><img src="img/vectorstock_23650232.jpg" width="500" /></p>
<p>This GitBook is under development (2024).</p>
<p>This GitBook provides a basic step-by-step protocol for students who want to get started with sequencing multiple samples of high quality, full-length microbial amplicons with an Oxford Nanopore instrument, and then process and visualise the data using various packages in R.</p>
<p>This protocol reflects the experiences we had in the lab and may help to get you started on your own journey of discovery. One of the biggest hurdles for students that embark on sequencing environmental DNA for the first time, is the effort required to learn new technologies, various coding languages, file types and formats, packages or platforms that are involved (think Unix, Linux, Slurm, Qiime, R, RMarkdown, python, conda, ggplot, docker, GitHub, data instances…) before they even can start looking at exploring the data for biological meaning and producing publishable output. Hence, this protocol aims at lowering your ‘activation’ energy by providing some guidance for each step from DNA extraction, to amplification, library preparation, sequencing, data processing and finally some basic visualisations in R.</p>
<p>This protocol uses the DNeasy Powersoil Pro Kit (Qiagen, Hilden, Germany) to extract DNA from wastewater sludge, as well as the Native Barcoding Kit (SQK-NBD114.96) with a R10.4.1 flowcell (FLO-MIN114) to sequence long (~4-4.4 kb) amplicons from environmental DNA. The primer used was developed by <span class="citation">(<a href="#ref-Martijn2019">Martijn et al. 2019</a>)</span>. Initial costs to purchase all consumables will be ~AUD$7,700.</p>
<p>Check out our workflows for short-read 16S sequencing at <a href="https://chrismitbiz.github.io/ABlab-workflows">https://chrismitbiz.github.io/ABlab-workflows</a> if you want to get started with Miseq 16S sequencing instead.</p>
<p>If you are interested in sequencing the living biomass via PMA treatment in combination with short-read 16S sequencing, check out our recent paper <a href="https://doi.org/10.1016/j.watres.2024.121354">Dead in the water – Role of relic DNA and primer choice for targeted sequencing surveys of anaerobic sewage sludge intended for biological monitoring</a> <span class="citation">(<a href="#ref-Krohn2024">Krohn et al. 2024</a>)</span></p>
<p><strong>Get in touch</strong></p>
<p>We work at the Andy Ball lab, RMIT University, Bundoora, Melbourne and are part of the Industry-led <a href="https://www.transformingbiosolids.org.au">Biosolids Training Centre</a>. Email me or comment on the discussion section of the <a href="https://github.com/chrismitbiz/ABlab-workflows/discussions/" target="_blank">GitHub repository</a> for this GitBook. You will need to get a GitHub account to join the discussion. Its free.</p>
<p>More about me and my PhD research can be found here: <a href="https://clean-dirt-digests.netlify.app" class="uri">https://clean-dirt-digests.netlify.app</a>.<br />
Follow me on <a href="https://twitter.com/CleanDirtChris">Twitter</a> or <a href="https://www.linkedin.com/in/christian-krohn-54904855">LinkedIn</a>.</p>
<p> </p>
<p><img src="img/avatar.jpg" width="150" /></p>
<p>Bio:<br />
Dr Chris Krohn is an early career researcher whose interests could be summed up with: “Environmental sequencing, microbial ecology, chlorinated pollutants, organic matter, wastewater, anaerobic digestion, and how everything connects”.</p>
<p>In 2021 I joined the ARC Biosolids Training Centre at RMIT (www.transformingbiosolids.org.au), where we help water utilities to improve circular resource management by getting more renewable biogas and carbon/fertiliser values out of our municipal biosolids (essentially our poo). In project 1C of the Centre I develop metagenomic (DNA-based) methods to monitor the microbiome of anaerobic digestion, an important microbial treatment process for wastewater. I believe DNA-based diagnoses of wastewater sludges will help the water/biosolids sector improve resource recoveries and risk management.</p>
<p>Before that, after a career in one of the most fast-cycled and short-sighted manufacturing industries that took me from Germany to Vietnam and Hong Kong/Shenzhen, I decided to hit the switch and start thinking long-term and circular. Ten back-to-uni years later, in 2021 I finished a PhD in Soil Science at La Trobe Uni where I sequenced soil DNA and explored if and how soil biology was involved in the degradation of extremely persistent legacy pesticides that contaminate agricultural surface soils since several decades.<br />
</p>
<p><a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img src="https://i.creativecommons.org/l/by/4.0/88x31.png" alt="Creative Commons License" style="border-width:0"/></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.</p>

</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-Krohn2024" class="csl-entry">
Krohn, Christian, Kraiwut Jansriphibul, Daniel A. Dias, Catherine A. Rees, Ben van den Akker, Jennifer C. Boer, Magdalena Plebanski, et al. 2024. <span>“<span class="nocase">Dead in the water – Role of relic DNA and primer choice for targeted sequencing surveys of anaerobic sewage sludge intended for biological monitoring</span>.”</span> <em>Water Research</em> 253: 121354. <a href="https://doi.org/10.1016/j.watres.2024.121354">https://doi.org/10.1016/j.watres.2024.121354</a>.
</div>
<div id="ref-Martijn2019" class="csl-entry">
Martijn, Joran, Anders E. Lind, Max E. Schön, Ian Spiertz, Lina Juzokaite, Ignas Bunikis, Olga V. Pettersson, and Thijs J. G. Ettema. 2019. <span>“<span class="nocase">Confident phylogenetic identification of uncultured prokaryotes through long read amplicon sequencing of the 16S-ITS-23S rRNA operon</span>.”</span> <em>Environmental Microbiology</em> 21 (7): 2485–98. <a href="https://doi.org/10.1111/1462-2920.14636">https://doi.org/10.1111/1462-2920.14636</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>

<a href="prerequisites.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/index.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ABlab-workflows-longreads.pdf", "ABlab-workflows-longreads.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
