<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 Prerequisites, equipment and consumables | A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</title>
  <meta name="description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  <meta name="generator" content="bookdown 0.40 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 Prerequisites, equipment and consumables | A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  <meta name="github-repo" content="ABlab-workflows-longreads" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 Prerequisites, equipment and consumables | A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]" />
  
  <meta name="twitter:description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  

<meta name="author" content="Christian Krohn, PhD, RMIT University" />


<meta name="date" content="2024-09-10" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="index.html"/>
<link rel="next" href="protocol.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Andy Ball lab RMIT</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> About this GitBook</a></li>
<li class="chapter" data-level="2" data-path="prerequisites.html"><a href="prerequisites.html"><i class="fa fa-check"></i><b>2</b> Prerequisites, equipment and consumables</a>
<ul>
<li class="chapter" data-level="2.1" data-path="prerequisites.html"><a href="prerequisites.html#prerequisites-1"><i class="fa fa-check"></i><b>2.1</b> Prerequisites</a></li>
<li class="chapter" data-level="2.2" data-path="prerequisites.html"><a href="prerequisites.html#dna-extraction-kit"><i class="fa fa-check"></i><b>2.2</b> DNA extraction kit</a></li>
<li class="chapter" data-level="2.3" data-path="prerequisites.html"><a href="prerequisites.html#other-consumables"><i class="fa fa-check"></i><b>2.3</b> Other consumables</a></li>
<li class="chapter" data-level="2.4" data-path="prerequisites.html"><a href="prerequisites.html#protocols"><i class="fa fa-check"></i><b>2.4</b> Protocols</a></li>
<li class="chapter" data-level="2.5" data-path="prerequisites.html"><a href="prerequisites.html#equipment"><i class="fa fa-check"></i><b>2.5</b> Equipment</a></li>
<li class="chapter" data-level="2.6" data-path="prerequisites.html"><a href="prerequisites.html#computational-resources-and-database"><i class="fa fa-check"></i><b>2.6</b> Computational resources and database</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="protocol.html"><a href="protocol.html"><i class="fa fa-check"></i><b>3</b> Protocol</a>
<ul>
<li class="chapter" data-level="3.1" data-path="protocol.html"><a href="protocol.html#dna-extraction"><i class="fa fa-check"></i><b>3.1</b> DNA extraction</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="protocol.html"><a href="protocol.html#checklist"><i class="fa fa-check"></i><b>3.1.1</b> Checklist</a></li>
<li class="chapter" data-level="3.1.2" data-path="protocol.html"><a href="protocol.html#steps"><i class="fa fa-check"></i><b>3.1.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="protocol.html"><a href="protocol.html#amplification-of-16s-its-23-operon"><i class="fa fa-check"></i><b>3.2</b> Amplification of 16S-ITS-23 operon</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="protocol.html"><a href="protocol.html#checklist-1"><i class="fa fa-check"></i><b>3.2.1</b> Checklist</a></li>
<li class="chapter" data-level="3.2.2" data-path="protocol.html"><a href="protocol.html#pcr"><i class="fa fa-check"></i><b>3.2.2</b> PCR</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="protocol.html"><a href="protocol.html#pcr-product-clean-up"><i class="fa fa-check"></i><b>3.3</b> PCR product clean-up</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="protocol.html"><a href="protocol.html#checklist-2"><i class="fa fa-check"></i><b>3.3.1</b> Checklist</a></li>
<li class="chapter" data-level="3.3.2" data-path="protocol.html"><a href="protocol.html#steps-1"><i class="fa fa-check"></i><b>3.3.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="protocol.html"><a href="protocol.html#library-preparation"><i class="fa fa-check"></i><b>3.4</b> Library preparation</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="protocol.html"><a href="protocol.html#checklist-3"><i class="fa fa-check"></i><b>3.4.1</b> Checklist</a></li>
<li class="chapter" data-level="3.4.2" data-path="protocol.html"><a href="protocol.html#steps-2"><i class="fa fa-check"></i><b>3.4.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="protocol.html"><a href="protocol.html#priming-and-loading-of-flow-cell---run-start"><i class="fa fa-check"></i><b>3.5</b> Priming and loading of flow cell - run start</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="protocol.html"><a href="protocol.html#checklist-4"><i class="fa fa-check"></i><b>3.5.1</b> Checklist</a></li>
<li class="chapter" data-level="3.5.2" data-path="protocol.html"><a href="protocol.html#steps-3"><i class="fa fa-check"></i><b>3.5.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="protocol.html"><a href="protocol.html#flow-cell-wash"><i class="fa fa-check"></i><b>3.6</b> Flow cell wash</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="protocol.html"><a href="protocol.html#checklist-5"><i class="fa fa-check"></i><b>3.6.1</b> Checklist</a></li>
<li class="chapter" data-level="3.6.2" data-path="protocol.html"><a href="protocol.html#steps-4"><i class="fa fa-check"></i><b>3.6.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="protocol.html"><a href="protocol.html#basecalling"><i class="fa fa-check"></i><b>3.7</b> Basecalling</a>
<ul>
<li class="chapter" data-level="3.7.1" data-path="protocol.html"><a href="protocol.html#checklist-6"><i class="fa fa-check"></i><b>3.7.1</b> Checklist</a></li>
<li class="chapter" data-level="3.7.2" data-path="protocol.html"><a href="protocol.html#steps-5"><i class="fa fa-check"></i><b>3.7.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.8" data-path="protocol.html"><a href="protocol.html#read-classification"><i class="fa fa-check"></i><b>3.8</b> Read classification</a>
<ul>
<li class="chapter" data-level="3.8.1" data-path="protocol.html"><a href="protocol.html#checklist-7"><i class="fa fa-check"></i><b>3.8.1</b> Checklist</a></li>
<li class="chapter" data-level="3.8.2" data-path="protocol.html"><a href="protocol.html#steps-6"><i class="fa fa-check"></i><b>3.8.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.9" data-path="protocol.html"><a href="protocol.html#results"><i class="fa fa-check"></i><b>3.9</b> Results</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="prerequisites" class="section level1 hasAnchor" number="2">
<h1><span class="header-section-number">Chapter 2</span> Prerequisites, equipment and consumables<a href="prerequisites.html#prerequisites" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="prerequisites-1" class="section level2 hasAnchor" number="2.1">
<h2><span class="header-section-number">2.1</span> Prerequisites<a href="prerequisites.html#prerequisites-1" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Experience with basic molecular laboratory techniques and DNA quality control (use of laminar flow hood, pipetting micro volumes, handling and preserving small volume reagents, sterilisation of consumables, doing gel electrophoresis)</li>
<li>Experience with loading Nanopore flowcells
<ul>
<li>It is vital that loading of the flowcell is practiced.The key is to avoid any air entering the flow cell as that will completely destroy the pores. Use an old flowcell if available and run through the process with water to see where the buffers flow. Afterwards we recommend doing a Control run with Lambda DNA using the <a href="https://store.nanoporetech.com/control-expansion.html">Control Expansion Kit</a>).</li>
</ul></li>
<li>Confident with R, R Studio and package environment managers such as Conda and Docker. Knowledge in shell scripting and Linux syntax.</li>
</ul>
</div>
<div id="dna-extraction-kit" class="section level2 hasAnchor" number="2.2">
<h2><span class="header-section-number">2.2</span> DNA extraction kit<a href="prerequisites.html#dna-extraction-kit" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>We commonly extract DNA from soils or from wastewater sludges using the DNeasy Powersoil Pro Kit (Qiagen) for both, soil and sludge. It resulted in high quality DNA, suitable for library preparation and sequencing. Extraction with this kit includes a bead beating step to release DNA from cells out of difficult matrices such as soils and sludge (biofilms) or similar. The sheared DNA will be fragmented with a N50 of ~7 kb in some cases <span class="citation">(<a href="#ref-Jensen2024">Jensen et al. 2024</a>)</span>. Shearing of DNA may be minimised by reducing/optimising bead-beating duration and speed. Other commonly used and commercially available kits for DNA extraction are available <span class="citation">(<a href="#ref-Jensen2024">Jensen et al. 2024</a>; <a href="#ref-Gand2023">Gand et al. 2023</a>)</span>.</p>
<div class="float">
<img src="img/readsize.png" width="600" alt="“Example of read size distribution of sludge DNA extracted with the DNeasy Powersoil Pro Kit in our lab”" />
<div class="figcaption">“Example of read size distribution of sludge DNA extracted with the DNeasy Powersoil Pro Kit in our lab”</div>
</div>
</div>
<div id="other-consumables" class="section level2 hasAnchor" number="2.3">
<h2><span class="header-section-number">2.3</span> Other consumables<a href="prerequisites.html#other-consumables" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Native Barcoding Kit (Oxford Nanopore, SQK-NBD114.96)<br />
</li>
<li>R10.4.1 flowcell (Oxford Nanopore, FLO-MIN114)</li>
<li>NEBNext Quick Ligation Module (E6056, New England Biolab)</li>
<li>NEBNext Ultra II End repair/dA-tailing Module (E7546S, New England Biolab)</li>
<li>Blunt/TA Ligase Master Mix (M0367, New England Biolab)</li>
<li>10 mM dNTPs (N0447S, New England Biolab)</li>
<li>Q5 Hot Start High-Fidelity DNA Polymerase (M0493, New England Biolab)</li>
<li>DNeasy Powersoil Pro Kit (#47016, Qiagen)</li>
<li>Qubit 1X dsDNA BR or HR Assay Kit (Q33266 or Q33231, Thermo Fisher)</li>
<li>Eppendorf LoBind tubes (Eppendorf)</li>
<li>twin.tec® PCR plate 96 LoBind, semi skirted (0030129504, Eppendorf)</li>
<li>JetSeq Clean Magnetic beads (MER-BIO-68031, Millenium Science) - to clean up PCR products</li>
<li>10 µM Forward Primer A519F (CAGCMGCCGCGGTAA) <span class="citation">(<a href="#ref-Martijn2019">Martijn et al. 2019</a>)</span></li>
<li>10 µM Reverse Primer U2428R (CCRAMCTGTCTCACGACG) <span class="citation">(<a href="#ref-Martijn2019">Martijn et al. 2019</a>)</span></li>
<li>Nuclease-free water</li>
<li>10 mM Tris-HCl pH 8.0 with 50 mM NaCl (UltraPure™ 1M Tris-HCI, pH 8.0 #15568025 &amp; NaCl (5 M), RNase-free #AM9760G, Thermo Fisher)</li>
<li>Bovine Serum Albumin (BSA) (50 mg/ml) (AM2616, InvitrogenTM UltraPure)</li>
<li>HyperLadder 1kb (BIO-33025, Bioline, Millenium Science)</li>
<li>80% ethanol, freshly prepared in nuclease-free water</li>
<li>Flow Cell Wash Kit (e.g. EXP-WSH004, Oxford Nanopore)</li>
</ul>
</div>
<div id="protocols" class="section level2 hasAnchor" number="2.4">
<h2><span class="header-section-number">2.4</span> Protocols<a href="prerequisites.html#protocols" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>A protocol for PCR amplification and clean-up (included in this Gitbook).</li>
<li><code>Ligation sequencing amplicons - Native Barcoding Kit 96 V14</code> available on nanoporetech.com.</li>
<li>Prepare spreadsheets to normalise DNA into 96-well plates to avoid pipetting errors.</li>
<li><code>Flow Cell Wash Kit (EXP-WSH004 or EXP-WSH004-XL)-minion.pdf</code> available on nanopore.com.</li>
</ul>
</div>
<div id="equipment" class="section level2 hasAnchor" number="2.5">
<h2><span class="header-section-number">2.5</span> Equipment<a href="prerequisites.html#equipment" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li>Nanopore sequencer. This protocol is based on the MinION Mk1C but is not limited to this model.</li>
<li>A Linux computer or server instance for basecalling nanopore <code>.pod5</code> files at super high accuracy (SUP) (unless you own the Promethion P2i which has sufficient GPU power to keep up with SUP basecalling). More info below.</li>
<li>Nanodrop spectrophotometer to check DNA quality.</li>
<li>Qubit fluorometer (Thermo Fisher).</li>
<li>PCR thermal cycler.</li>
<li>Magnets for 96-well plates and 1.5 ml tubes.</li>
<li>Gel electrophoresis equipment.</li>
<li>Vortex. We use the Vortex-Genie 2, including the 24 x 2 ml tube adapter for bead beating.</li>
<li>Hula mixer (#15920D, Thermo Fisher) or similar overhead mixers.</li>
<li>Eppendorf LoBind tubes (Eppendorf)</li>
</ul>
</div>
<div id="computational-resources-and-database" class="section level2 hasAnchor" number="2.6">
<h2><span class="header-section-number">2.6</span> Computational resources and database<a href="prerequisites.html#computational-resources-and-database" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li><p>For basecalling the raw nanopore data (pod5 files) with dorado - a Linux instance or computer is required with ≥ 2 TB storage, 64 GB DDR4 RAM and at least one Nvidia A100 GPU (e.g. GeForce RTX 4090). Note that the MinION Mk1C or the GridION can also basecall the data (much easier). But in the case of the MinION only at High Accuracy (HAC) and not at Super High Accuracy (SUP). But even at HAC it takes days to finish. Note that SUP achieves ~Q20-Q25 while HAC only ~Q18.</p></li>
<li><p>16S-ITS-23S Database. To be confirmed.</p></li>
</ul>
<div class="float">
<img src="img/basecallsummary.png" width="400" alt="“Example of quality distribution of 4kb amplicon reads basecalled at super high accuracy with dorado.”" />
<div class="figcaption">“Example of quality distribution of 4kb amplicon reads basecalled at super high accuracy with dorado.”</div>
</div>
<div class="float">
<img src="img/basecallsummaryHAC.png" width="400" alt="“Example of quality distribution of 4kb amplicon reads basecalled at high accuracy with dorado.”" />
<div class="figcaption">“Example of quality distribution of 4kb amplicon reads basecalled at high accuracy with dorado.”</div>
</div>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-Gand2023" class="csl-entry">
Gand, Mathieu, Bram Bloemen, Kevin Vanneste, Nancy H. C. Roosens, and Sigrid C. J. De Keersmaecker. 2023. <span>“<span class="nocase">Comparison of 6 DNA extraction methods for isolation of high yield of high molecular weight DNA suitable for shotgun metagenomics Nanopore sequencing to detect bacteria</span>.”</span> <em>BMC Genomics</em> 24 (1): 438. <a href="https://doi.org/10.1186/s12864-023-09537-5">https://doi.org/10.1186/s12864-023-09537-5</a>.
</div>
<div id="ref-Jensen2024" class="csl-entry">
Jensen, Thomas Bygh Nymann, Sebastian Mølvang Dall, Simon Knutsson, Søren Michael Karst, and Mads Albertsen. 2024. <span>“<span class="nocase">High-throughput DNA extraction and cost-effective miniaturized metagenome and amplicon library preparation of soil samples for DNA sequencing</span>.”</span> <em>PLoS ONE</em> 19 (4 April): e0301446. <a href="https://doi.org/10.1371/journal.pone.0301446">https://doi.org/10.1371/journal.pone.0301446</a>.
</div>
<div id="ref-Martijn2019" class="csl-entry">
Martijn, Joran, Anders E. Lind, Max E. Schön, Ian Spiertz, Lina Juzokaite, Ignas Bunikis, Olga V. Pettersson, and Thijs J. G. Ettema. 2019. <span>“<span class="nocase">Confident phylogenetic identification of uncultured prokaryotes through long read amplicon sequencing of the 16S-ITS-23S rRNA operon</span>.”</span> <em>Environmental Microbiology</em> 21 (7): 2485–98. <a href="https://doi.org/10.1111/1462-2920.14636">https://doi.org/10.1111/1462-2920.14636</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="index.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="protocol.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/02-prerequisites.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ABlab-workflows-longreads.pdf", "ABlab-workflows-longreads.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
