<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 3 Protocol | A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</title>
  <meta name="description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  <meta name="generator" content="bookdown 0.39 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 3 Protocol | A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  <meta name="github-repo" content="ABlab-workflows-longreads" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 3 Protocol | A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]" />
  
  <meta name="twitter:description" content="A protocol for novel speciec-level detection of thousands of bacteria and archaea." />
  

<meta name="author" content="Christian Krohn, PhD, RMIT University" />


<meta name="date" content="2024-06-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="prerequisites.html"/>
<link rel="next" href="references.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>



<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Andy Ball lab RMIT</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> About this GitBook</a></li>
<li class="chapter" data-level="2" data-path="prerequisites.html"><a href="prerequisites.html"><i class="fa fa-check"></i><b>2</b> Prerequisites, equipment and consumables</a>
<ul>
<li class="chapter" data-level="2.1" data-path="prerequisites.html"><a href="prerequisites.html#prerequisites-1"><i class="fa fa-check"></i><b>2.1</b> Prerequisites</a></li>
<li class="chapter" data-level="2.2" data-path="prerequisites.html"><a href="prerequisites.html#dna-extraction-kit"><i class="fa fa-check"></i><b>2.2</b> DNA extraction kit</a></li>
<li class="chapter" data-level="2.3" data-path="prerequisites.html"><a href="prerequisites.html#other-consumables"><i class="fa fa-check"></i><b>2.3</b> Other consumables</a></li>
<li class="chapter" data-level="2.4" data-path="prerequisites.html"><a href="prerequisites.html#protocols"><i class="fa fa-check"></i><b>2.4</b> Protocols</a></li>
<li class="chapter" data-level="2.5" data-path="prerequisites.html"><a href="prerequisites.html#equipment"><i class="fa fa-check"></i><b>2.5</b> Equipment</a></li>
<li class="chapter" data-level="2.6" data-path="prerequisites.html"><a href="prerequisites.html#computational-resources-and-database"><i class="fa fa-check"></i><b>2.6</b> Computational resources and database</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="protocol.html"><a href="protocol.html"><i class="fa fa-check"></i><b>3</b> Protocol</a>
<ul>
<li class="chapter" data-level="3.1" data-path="protocol.html"><a href="protocol.html#dna-extraction"><i class="fa fa-check"></i><b>3.1</b> DNA extraction</a>
<ul>
<li class="chapter" data-level="3.1.1" data-path="protocol.html"><a href="protocol.html#checklist"><i class="fa fa-check"></i><b>3.1.1</b> Checklist</a></li>
<li class="chapter" data-level="3.1.2" data-path="protocol.html"><a href="protocol.html#steps"><i class="fa fa-check"></i><b>3.1.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.2" data-path="protocol.html"><a href="protocol.html#amplification-of-16s-its-23-operon"><i class="fa fa-check"></i><b>3.2</b> Amplification of 16S-ITS-23 operon</a>
<ul>
<li class="chapter" data-level="3.2.1" data-path="protocol.html"><a href="protocol.html#checklist-1"><i class="fa fa-check"></i><b>3.2.1</b> Checklist</a></li>
<li class="chapter" data-level="3.2.2" data-path="protocol.html"><a href="protocol.html#pcr"><i class="fa fa-check"></i><b>3.2.2</b> PCR</a></li>
</ul></li>
<li class="chapter" data-level="3.3" data-path="protocol.html"><a href="protocol.html#pcr-product-clean-up"><i class="fa fa-check"></i><b>3.3</b> PCR product clean-up</a>
<ul>
<li class="chapter" data-level="3.3.1" data-path="protocol.html"><a href="protocol.html#checklist-2"><i class="fa fa-check"></i><b>3.3.1</b> Checklist</a></li>
<li class="chapter" data-level="3.3.2" data-path="protocol.html"><a href="protocol.html#steps-1"><i class="fa fa-check"></i><b>3.3.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.4" data-path="protocol.html"><a href="protocol.html#library-preparation"><i class="fa fa-check"></i><b>3.4</b> Library preparation</a>
<ul>
<li class="chapter" data-level="3.4.1" data-path="protocol.html"><a href="protocol.html#checklist-3"><i class="fa fa-check"></i><b>3.4.1</b> Checklist</a></li>
<li class="chapter" data-level="3.4.2" data-path="protocol.html"><a href="protocol.html#steps-2"><i class="fa fa-check"></i><b>3.4.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.5" data-path="protocol.html"><a href="protocol.html#priming-and-loading-of-flow-cell---run-start"><i class="fa fa-check"></i><b>3.5</b> Priming and loading of flow cell - run start</a>
<ul>
<li class="chapter" data-level="3.5.1" data-path="protocol.html"><a href="protocol.html#checklist-4"><i class="fa fa-check"></i><b>3.5.1</b> Checklist</a></li>
<li class="chapter" data-level="3.5.2" data-path="protocol.html"><a href="protocol.html#steps-3"><i class="fa fa-check"></i><b>3.5.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.6" data-path="protocol.html"><a href="protocol.html#flow-cell-wash"><i class="fa fa-check"></i><b>3.6</b> Flow cell wash</a>
<ul>
<li class="chapter" data-level="3.6.1" data-path="protocol.html"><a href="protocol.html#checklist-5"><i class="fa fa-check"></i><b>3.6.1</b> Checklist</a></li>
<li class="chapter" data-level="3.6.2" data-path="protocol.html"><a href="protocol.html#steps-4"><i class="fa fa-check"></i><b>3.6.2</b> Steps</a></li>
</ul></li>
<li class="chapter" data-level="3.7" data-path="protocol.html"><a href="protocol.html#basecalling"><i class="fa fa-check"></i><b>3.7</b> Basecalling</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A protocol for sequencing and analysing 16S-ITS-23S amplicons using Oxford Nanopore’s Native Barcoding kit to profile prokaryotes on species-level in a mixed community [Under development]</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="protocol" class="section level1 hasAnchor" number="3">
<h1><span class="header-section-number">Chapter 3</span> Protocol<a href="protocol.html#protocol" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="dna-extraction" class="section level2 hasAnchor" number="3.1">
<h2><span class="header-section-number">3.1</span> DNA extraction<a href="protocol.html#dna-extraction" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="checklist" class="section level3 hasAnchor" number="3.1.1">
<h3><span class="header-section-number">3.1.1</span> Checklist<a href="protocol.html#checklist" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Extraction kit - <a href="https://www.qiagen.com/au/resources/resourcedetail?id=9bb59b74-e493-4aeb-b6c1-f660852e8d97&amp;lang=en">DNeasy Powersoil Pro, Qiagen, Hilden, Germany</a>. See Section <a href="prerequisites.html#prerequisites">2</a> for more details.</li>
<li>Alternatively the high throughput version of the kit used with the QIAcube <a href="https://rmiteduau-my.sharepoint.com/personal/christian_krohn_rmit_edu_au/Documents/Experiments/2025_ETPSequencingProject/_%09https:/www.qiagen.com/au/products/discovery-and-translational-research/dna-rna-purification/dna-purification/microbial-dna/dneasy-96-powersoil-pro-qiacube-ht-kit">DNeasy 96 PowerSoil Pro QIAcube HT Kit</a>.</li>
<li>Vortex with 24 x 1.5 mL tube adapter (e.g. Vortex Genie 2 + adapter). Alternatively, a PowerLyzer Homogenizer.</li>
<li>NanoDrop spectrophotometer to assess DNA quality.</li>
<li>Qubit fluorometer to accurately measure DNA concentrations.</li>
</ul>
</div>
<div id="steps" class="section level3 hasAnchor" number="3.1.2">
<h3><span class="header-section-number">3.1.2</span> Steps<a href="protocol.html#steps" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Follow the extraction kit’s protocol with 15 mins bead beating using a Vortex-Genie and 24-tube adapter. Reduce this to 10 mins if less than 24 samples are extracted or if shearing of DNA should be minimised.</li>
<li>Measure DNA quality using DNA extract (1 µL) using a Nanodrop spectrophotometer</li>
<li>Measure DNA concentration using a Qubit fluorometer.</li>
<li>Into a 96-well plate, normalise extracted DNA to required PCR concentrations. For example, if 10 ng template is required for PCR, normalise DNA to 5 ng/µL.</li>
<li>Store DNA at 4˚C until library preparation - no more than 1 week.</li>
<li>Store DNA at -20˚C if sequencing is more than 1 week away.</li>
</ul>
</div>
</div>
<div id="amplification-of-16s-its-23-operon" class="section level2 hasAnchor" number="3.2">
<h2><span class="header-section-number">3.2</span> Amplification of 16S-ITS-23 operon<a href="protocol.html#amplification-of-16s-its-23-operon" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<div id="checklist-1" class="section level3 hasAnchor" number="3.2.1">
<h3><span class="header-section-number">3.2.1</span> Checklist<a href="protocol.html#checklist-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>10 mM dNTPs (N0447S, New England Biolab)</li>
<li>Q5 Hot Start High-Fidelity DNA Polymerase (M0493, New England Biolab)</li>
<li>10 µM Forward Primer A519F (CAGCMGCCGCGGTAA) <span class="citation">(<a href="#ref-Martijn2019">Martijn et al. 2019</a>)</span></li>
<li>10 µM Reverse Primer U2428R (CCRAMCTGTCTCACGACG) <span class="citation">(<a href="#ref-Martijn2019">Martijn et al. 2019</a>)</span></li>
<li>JetSeq Clean Magnetic beads - or equivalent (MER-BIO-68031, Millenium Science) - to clean up PCR products</li>
<li>twin.tec® PCR plate 96 LoBind, semi skirted (0030129504, Eppendorf)</li>
<li>Nuclease-free water</li>
<li>10 mM Tris-HCl pH 8.0 with 50 mM NaCl (UltraPure™ 1M Tris-HCI, pH 8.0 #15568025 &amp; NaCl (5 M), RNase-free #AM9760G, Thermo Fisher)</li>
<li>HyperLadder 1kb (BIO-33025, Bioline, Millenium Science)</li>
<li>80% ethanol, freshly prepared in nuclease-free water</li>
<li>Qubit 1X dsDNA BR Assay Kit (Q33266, Thermo Fisher)</li>
</ul>
<p><strong>Notes</strong></p>
<ul>
<li>Do not vortex tubes during library preparation to prevent DNA fragmentation. Fragmentation of amplicons may lead to incomplete reads.</li>
<li>The primer amplifies the whole rrn operon.</li>
</ul>
<p>Benefits of targeting the whole rrn operon:</p>
<ul>
<li>Superior species-level resolution and accuracy <span class="citation">(<a href="#ref-Cusco2018">Cuscó et al. 2019</a>; <a href="#ref-Srinivas2024">Srinivas et al. 2024</a>)</span>.</li>
<li>Covers Bacteria and Archaea.</li>
</ul>
<p>Risks of targeting the whole rrn operon:</p>
<ul>
<li>Not as representative of true abundances as full-length 16S amplicons.</li>
<li>Species with unlinked 16S and 23S rrn DNA will be missed with this approach (for example &lt; 9% of rRNA genes in wastewater sludge <span class="citation">(<a href="#ref-Brewer2020">Brewer et al. 2020</a>)</span>).</li>
</ul>
</div>
<div id="pcr" class="section level3 hasAnchor" number="3.2.2">
<h3><span class="header-section-number">3.2.2</span> PCR<a href="protocol.html#pcr" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Time required: ~4 hrs incl. 2hrs, 40mins PCR.</p>
<ul>
<li>Prepare a PCR mastermix for the required number of 50-µl reactions (Table 3.1). It may be necessary to combine two 50 µl reactions for each sample to produce sufficient amplicon mass for a final concentration of 200 fmol as input for the Native Barcoding Kit from Oxford Nanopore.</li>
<li>Add 3 µl of eDNA (5ng /µl) into a 96-well plate (e.g. Eppendorf twin.tec® PCR plate 96 LoBind, semi-skirted) using a multichannel pipette.</li>
<li>Add 47 µl of mastermix using a multichannel pipette and carefully pipette up and down 10 x</li>
<li>Run thermocycler (Table 3.2).</li>
<li>Verify amplification length via 1% agarose gel electrophoresis @ 100V for 30 min including a 1 kb ladder.</li>
<li>Store at 4˚C overnight if needed.</li>
</ul>
<p><strong>Important</strong>:</p>
<ul>
<li><p>Use hot start polymerase for ease of use, with high fidelity/accuracy and one that is suitable for long amplicons. For example, Q5 High- Fidelity DNA Polymerase kit (New England Biolabs) with GC enhancer <span class="citation">(<a href="#ref-Martijn2019">Martijn et al. 2019</a>)</span>.</p></li>
<li><p>≤ 200 fmol is required per sample for the Native Barcoding Kit from ONP. Based on 4.25 kb (4-4.5kb), the final DNA concentration after cleaning up PCR products should be no less than 48 ng/µL (at 11.5 µl input volume) - giving 552 ng of 4.25kb amplicons.</p></li>
<li><p>It may require two PCR reactions to achieve the required DNA amount (200 fmol); e.g. pool 2 x 50µl PCR products, clean combined and elude in 32.5 µl Tris.</p>
<p>   </p></li>
</ul>
<table>
<caption><span id="tab:table">Table 3.1: </span>List of components for each reaction (each tube) for PCR. See <a href="https://www.neb.com/en-au/protocols/2012/08/30/pcr-using-q5-hot-start-high-fidelity-dna-polymerase-m0493" class="uri">https://www.neb.com/en-au/protocols/2012/08/30/pcr-using-q5-hot-start-high-fidelity-dna-polymerase-m0493</a> for details.</caption>
<colgroup>
<col width="60%" />
<col width="16%" />
<col width="22%" />
</colgroup>
<thead>
<tr class="header">
<th align="left">Component</th>
<th align="left">50 µl reaction</th>
<th align="left">Final concentration</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">5X Q5 Reaction Buffer - M0493S NEB</td>
<td align="left">10 µL</td>
<td align="left">1X</td>
</tr>
<tr class="even">
<td align="left">10 mM dNTPs N0447S</td>
<td align="left">1 µL</td>
<td align="left">200 µM</td>
</tr>
<tr class="odd">
<td align="left">10 µM Forward Primer</td>
<td align="left">2.5 µL</td>
<td align="left">0.5 µM</td>
</tr>
<tr class="even">
<td align="left">10 µM Reverse Primer</td>
<td align="left">2.5 µl</td>
<td align="left">0.5 µM</td>
</tr>
<tr class="odd">
<td align="left">Template DNA - 15 ng</td>
<td align="left">3 µL (5 ng/µL)</td>
<td align="left">&lt; 1,000 ng</td>
</tr>
<tr class="even">
<td align="left">Q5 Hot Start High-Fidelity DNA Polymerase -M0493S NEB</td>
<td align="left">1.0 µL</td>
<td align="left">0.04 U/µL</td>
</tr>
<tr class="odd">
<td align="left">5X Q5 High GC Enhancer M0493S NEB</td>
<td align="left">10 µL</td>
<td align="left">(1X)</td>
</tr>
<tr class="even">
<td align="left">Nuclease-Free Water</td>
<td align="left">to 50 µL</td>
<td align="left"></td>
</tr>
</tbody>
</table>
<p> </p>
<table>
<caption><span id="tab:table2">Table 3.2: </span>Thermocycler conditions (Martijn et al 2019).</caption>
<thead>
<tr class="header">
<th align="left">Cycle conditions</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="left">1 cycle:</td>
</tr>
<tr class="even">
<td align="left">30 s - Initial Denaturation 98 degree C</td>
</tr>
<tr class="odd">
<td align="left">35 cycles:</td>
</tr>
<tr class="even">
<td align="left">10 s - 98 degree C</td>
</tr>
<tr class="odd">
<td align="left">30 s - 64 degree C</td>
</tr>
<tr class="even">
<td align="left">210 s - 72 degree C</td>
</tr>
<tr class="odd">
<td align="left">1 cycle:</td>
</tr>
<tr class="even">
<td align="left">10 min - Final Extension 72 degree C</td>
</tr>
</tbody>
</table>
<p><br />
</p>
</div>
</div>
<div id="pcr-product-clean-up" class="section level2 hasAnchor" number="3.3">
<h2><span class="header-section-number">3.3</span> PCR product clean-up<a href="protocol.html#pcr-product-clean-up" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Clean with 0.6X JetSeq Clean Magnetic beads (or equivalent) and wash twice with 80% ethanol. This follows a similar protocol to the Illumina 16S-metagenomics library prep guide in case you are familiar with that.</p>
<p>Time required: ~ 1 hr per 24 samples.</p>
<div id="checklist-2" class="section level3 hasAnchor" number="3.3.1">
<h3><span class="header-section-number">3.3.1</span> Checklist<a href="protocol.html#checklist-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Magnetic rack for 96-well plate (e.g. #AM10027 Thermo Fisher)</li>
<li>80% ethanol, freshly prepared in nuclease-free water</li>
<li>Nuclease-free water</li>
<li>10 mM Tris-HCl pH 8.0 with 50 mM NaCl (UltraPure™ 1M Tris-HCI, pH 8.0 #15568025 &amp; NaCl (5 M), RNase-free #AM9760G, Thermo Fisher)</li>
<li>JetSeq Clean Magnetic beads (MER-BIO-68031, Millenium Science) or equivalent - to clean up PCR products</li>
<li>Qubit 1X dsDNA BR or HR Assay Kit (Q33266 or Q33231, Thermo Fisher)</li>
<li>HyperLadder 1kb (BIO-33025, Bioline, Millenium Science)</li>
<li>A worksheet to enter concentrations to calculate volumes to normalise DNA to 200 fmol</li>
</ul>
<p><strong>Note</strong></p>
<p>Use wide bore tips for adding mag beads and subsequent (careful) mixing by pipetting to minimise DNA fragmentation.</p>
</div>
<div id="steps-1" class="section level3 hasAnchor" number="3.3.2">
<h3><span class="header-section-number">3.3.2</span> Steps<a href="protocol.html#steps-1" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Pool replicate reactions into one well - e.g. 2 x 50 µl = 100 µl in one well.</li>
<li>Using a multi-channel pipette, add 0.6X beads to PCR products in the same wells (e.g. 60 µl to 100 µl pooled reactions), pipette 10 x up and down and/or a gentle vortex after sealing the plate, careful not to spill (Vortex Genie speed 2).</li>
<li>10 min incubation at RT</li>
<li>Place plate on a magnetic rack for 5 mins and remove supernatant.</li>
<li>Wash twice with 200 µl 80% ethanol in same 96-well plate, discard ethanol and let evaporate for ~ 5 mins or until completely removed.</li>
<li>Take plate off magnet and elude pellet to a volume that provides appropriate DNA concentrations
<ul>
<li>e.g. add 32.5 µl of elution buffer (10 mM Tris 50mM NaCl). The pellet will be stuck higher than the buffer surface in the well.</li>
<li>Hence, pipetting up and down 20–30 x may be necessary to carefully wash the pellet of the tube wall, using a wide bore tip.</li>
<li>And/or centrifuge plate at 200 rcf for 1 min.</li>
</ul></li>
<li>Off-magnet, incubate plate for 15 minutes at 37°C (e.g. in a water bath with only the wells submerged in water). Every 2 minutes, agitate the sample by gently vortexing the plate for 10 seconds (after sealing plate) to encourage DNA elution - and spin plate down.</li>
<li>Collect 32 µl of cleaned DNA after ~ 5 mins on magnet using wide bore tips.</li>
<li>Quantify and record amplicon concentrations using Qubit HS or BS chemistry and record values. Based on 4 kb amplicons, the final DNA concentration after cleaning up PCR products should be no less than 45 ng/µL to have sufficient material for downstream library preparation (200 fmol, &gt; 520 ng of 4 kb amplicons).</li>
<li>Verify amplification length via 0.5-1% agarose gel electrophoresis @ 100V for 30 min including a 1 kb ladder.</li>
<li>For each sample calculate volumes required of cleaned DNA and H<sub>2</sub>0 to get 200 fmol in 11.5 µl - for library preparation with the Native Barcoding Kit from Oxford Nanopore. Use V1 = C2*V2 / C1, where V1 = x, C1 = DNA concentrations (ng/µl), C2 = 45.22 ng/µl, V2 = 11.5 µl). H<sub>2</sub>0 volume = 11.5 µl - V1. These volumes will be used in the next step.</li>
</ul>
<p><br />
</p>
<div class="float">
<img src="img/PCR_gel.jpg" width="300" alt="Gel confirming amplification of ~4 kb amplicons" />
<div class="figcaption">Gel confirming amplification of ~4 kb amplicons</div>
</div>
<p><br />
</p>
<div class="float">
<img src="img/PCR_minDNA.png" width="400" alt="Volumes to normalise to 200 fmol (Example)." />
<div class="figcaption">Volumes to normalise to 200 fmol (Example).</div>
</div>
<p><br />
</p>
<div class="float">
<img src="img/PCR_C1V1.png" width="300" alt="Example of a worksheet for calculating volumes (µl) of PCR products and H20 to get 200 fmol in 11.5 µl, required as input for each sample with the Native Barcoding Kit. C1 = DNA concentration (ng/µl), C2 (library concentration) = 45.22 ng/µl, V2 (H20 volume) = 11.5 µl." />
<div class="figcaption">Example of a worksheet for calculating volumes (µl) of PCR products and H<sub>2</sub>0 to get 200 fmol in 11.5 µl, required as input for each sample with the Native Barcoding Kit. C1 = DNA concentration (ng/µl), C2 (library concentration) = 45.22 ng/µl, V2 (H<sub>2</sub>0 volume) = 11.5 µl.</div>
</div>
<p> </p>
</div>
</div>
<div id="library-preparation" class="section level2 hasAnchor" number="3.4">
<h2><span class="header-section-number">3.4</span> Library preparation<a href="protocol.html#library-preparation" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>DNA Library preparation, including barcoding and pooling of samples (multiplexing). Here the DNA input are &gt;4 kb amplicons of the 16S-ITS-23S rRNA operon from environmental DNA. The Long Fragment Buffer (LFB) is used (not the Short Fragment Buffer)</p>
<p>Time required: ~ 6 hours from start to finished library ready for loading.</p>
<p><strong>Notes</strong></p>
<ul>
<li>Follow the protocol <code>Ligation sequencing amplicons - Native Barcoding Kit 96 V14 (SQK-NBD114.96)-minion.pdf</code> available on nanoporetech.com. For details about the barcodes check here: <a href="https://community.nanoporetech.com/technical_documents/chemistry-technical-document/v/chtd_500_v1_revaq_07jul2016/barcode-sequences" class="uri">https://community.nanoporetech.com/technical_documents/chemistry-technical-document/v/chtd_500_v1_revaq_07jul2016/barcode-sequences</a>. For final library calculations we assume that around 300 bps are added to the amplicons through barcodes, flanking sequences and adapters. If you know more - let us know on the Github discussion section.</li>
<li>Update the MinION packages and MinKNOW software.</li>
<li>Prepare a sample sheet for upload to instrument if desired (not essential but saves a little time when setting up the sequencing in MinKNOW). More info on how to upload sample sheets <a href="https://community.nanoporetech.com/docs/prepare/library_prep_protocols/experiment-companion-minknow/v/mke_1013_v1_revdc_11apr2016/sample-sheet-upload">here</a>.</li>
<li>Amplicon product yields from PCR can vary widely depending on environmental source.</li>
<li>To achieve consistent, normalised concentrations (200 fmol) across all samples, add appropriate input volume (calculated with C1*V1 = C2*V2) and fill to 11.5 µl with H<sub>2</sub>0 to the first step of the Native Barcoding kit protocol.</li>
<li>If you plan to sequence after library preparations, insert a flow cell now and do a <strong>flow cell check</strong> and record the number of pores. This will take around 10 minutes. Afterwards store it at 4˚C until ready to load. We got 1,500-1,750 pores with new flowcells (R10.4.1) that were stored for two weeks at 4 degrees.</li>
</ul>
<div id="checklist-3" class="section level3 hasAnchor" number="3.4.1">
<h3><span class="header-section-number">3.4.1</span> Checklist<a href="protocol.html#checklist-3" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Native Barcoding Kit (Oxford Nanopore, SQK-NBD114.96)
<ul>
<li>End-prep
<ul>
<li>DNA Control Sample (DCS)</li>
<li>Elution Buffer (EB)</li>
</ul></li>
<li>Native barcode ligation
<ul>
<li>AMPure XP Beads (AXP)</li>
<li>EDTA</li>
<li>Native Barcodes (NB01-96)</li>
</ul></li>
<li>Sequencing adapter ligation
<ul>
<li>Native Adapter (NA)</li>
<li>Elution Buffer (EB)</li>
<li>AMPure XP Beads (AXP)</li>
<li>Long Fragment Buffer (LFB)</li>
</ul></li>
</ul></li>
<li>NEBNext Quick Ligation Module (E6056, New England Biolab)</li>
<li>NEBNext Ultra II End repair/dA-tailing Module (E7546S, New England Biolab)</li>
<li>Blunt/TA Ligase Master Mix (M0367, New England Biolab)</li>
<li>80% ethanol, freshly prepared in nuclease-free water</li>
<li>Nuclease-free water</li>
<li>Qubit 1X dsDNA BR or HR Assay Kit (Q33266 or Q33231, Thermo Fisher)</li>
<li>Eppendorf LoBind tubes (Eppendorf)</li>
<li>Magnet for 1.5 ml tubes</li>
<li>Hula mixer (#15920D, Thermo Fisher) or similar overhead mixers</li>
</ul>
</div>
<div id="steps-2" class="section level3 hasAnchor" number="3.4.2">
<h3><span class="header-section-number">3.4.2</span> Steps<a href="protocol.html#steps-2" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Follow the Nanopore protocol. There are four parts:</strong></p>
<ol style="list-style-type: decimal">
<li><p>End-prepping of 11.5 µL DNA (200 fmol).<br />
Pipetting into the 96-well plate may take time as there are two rounds of pipetting, one for the DNA amplicon volumes and one for H<sub>2</sub>0 volumes to get 200 fmol amplicon DNA in 11.5 µl, as well as the remaining reagents for end-prepping the DNA. It is not possible to use multichannel pipettes for this step.</p></li>
<li><p>Native barcode ligation using 0.75 µL of end-prepped DNA using one of 96 barcodes from the kit for each sample, plus pooling of all barcoded samples into a LoBind tube (1.5 mL).</p></li>
<li><p>Bead clean-up of pooled end-prepped DNA library and elution of 35 µL volume; quantification with Qubit 1X dsDNA HS Assay Kit. We used the all of the available end-prepped volume (e.g. 33 µl after Qubit quantification) for the next step, i.o. only 30 µl as per protocol.</p>
<p>SAFE STOP: Store at 4 ˚C overnight if needed.</p></li>
<li><p>Adapter ligation. Once this step is completed, you get a final volume of 15 µl of clean end-prepped/barcoded/adapter ligated DNA.</p>
<ul>
<li><p>We have stored the library at 4˚C overnight and sucessfully sequenced it the next day (although it is preferred to sequence right after adapter ligation).</p></li>
<li><p>The concentration we achieved was ~20-25 ng/µl, which means there was sufficient DNA for two runs. After quantify DNA concentrations with 1 µl DNA there are 14 µl left.</p></li>
<li><p>Nanopore’s recommended loading concentrations are 35–50 fmol at 12 µL volume. For example, at a concentration of 25.3 ng/µl (4,300 kb amplicon) a total of 5.50 µl of DNA library (plus 6.50 µl H<sub>2</sub>0) are sufficient. The remaining 6.50 µl DNA library can be stored in fridge overnight if required.</p></li>
</ul></li>
</ol>
<p><img src="img/barcode_ligation.png" width="300" alt="The Native Barcode Kit (SQK-NBD114.96) comes with 96 individual barcodes" /><br />
</p>
<p><img src="img/libraryprepcleanup_comb.png" width="500" alt="Bead clean-up of pooled end-prepped and adapter-ligated DNA library" /><br />
</p>
<div class="float">
<img src="img/libraryprepsminDNA.png" width="300" alt="Calculation to get library volume for loading (In this example we used 5.52 µl of library, plus 6.48µl H20 for a final library volume of 12 µl at 50 fmol concentration)" />
<div class="figcaption">Calculation to get library volume for loading (In this example we used 5.52 µl of library, plus 6.48µl H<sub>2</sub>0 for a final library volume of 12 µl at 50 fmol concentration)</div>
</div>
</div>
</div>
<div id="priming-and-loading-of-flow-cell---run-start" class="section level2 hasAnchor" number="3.5">
<h2><span class="header-section-number">3.5</span> Priming and loading of flow cell - run start<a href="protocol.html#priming-and-loading-of-flow-cell---run-start" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Time required: ~10-15 mins for priming and loading. 24 hours for sequencing.</p>
<div id="checklist-4" class="section level3 hasAnchor" number="3.5.1">
<h3><span class="header-section-number">3.5.1</span> Checklist<a href="protocol.html#checklist-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>The same protocol: <code>Ligation sequencing amplicons - Native Barcoding Kit 96 V14 (SQK-NBD114.96)-minion.pdf</code> available on nanoporetech.com</li>
<li>Bovine Serum Albumin (BSA) (50 mg/ml) (AM2616, InvitrogenTM UltraPure)</li>
<li>Native Barcoding Kit (Oxford Nanopore, SQK-NBD114.96)
<ul>
<li>Flow Cell Flush (FCF)</li>
<li>Flow Cell Tether (FCT)</li>
<li>Sequencing Buffer (SB)</li>
<li>Library Beads (LIB)</li>
</ul></li>
<li>R10.4.1 flowcell (Oxford Nanopore, FLO-MIN114)</li>
</ul>
</div>
<div id="steps-3" class="section level3 hasAnchor" number="3.5.2">
<h3><span class="header-section-number">3.5.2</span> Steps<a href="protocol.html#steps-3" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Follow the Nanopore protocol. There are five steps:</strong></p>
<ol style="list-style-type: decimal">
<li>Preparation and loading of sequencing buffer and wait for 5 minutes.</li>
<li>Preparation of library. Nanopore’s protocol recommends loading 35–50 fmol at 12 µL volume.</li>
<li>Loading of remaining sequencing buffer</li>
<li>Loading of library</li>
<li>Setting up the run for 24 hours and press start! More details will follow about recommended hours and expected yield based on 24 samples.</li>
</ol>
</div>
</div>
<div id="flow-cell-wash" class="section level2 hasAnchor" number="3.6">
<h2><span class="header-section-number">3.6</span> Flow cell wash<a href="protocol.html#flow-cell-wash" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Time required: ~ 1.5 hours.</p>
<p>Wash the flow cell once the run has finished. Note that after the wash you can either load a new library directly afterwards or add storage buffer to load a new library later (always store flow at 4 ˚C).</p>
<p>However, it is only possible to check the number of pores (and estimate potential yield) after adding storage buffer. The washing buffer is not suitable for a running a flow cell check. So loading a new library directly is only sensible if the user knows how many active pores to expect. Although, the instrument will do a pore scan at the start, it will be too late once you loaded the new library.</p>
<p>As a gauge, we ran a flowcell with 1,500 pores for 24 hours (loaded with 24 barcoded samples of 16S-ITS-23S amplicons). At the end 558 active pores remained before the wash.</p>
<p>Note that washing the flowcell does not interfer with any active basecalling processes on the Mk1C, which may take days to finish if High Accuracy was chosen.</p>
<div id="checklist-5" class="section level3 hasAnchor" number="3.6.1">
<h3><span class="header-section-number">3.6.1</span> Checklist<a href="protocol.html#checklist-5" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<ul>
<li>Flow Cell Wash Kit (e.g. EXP-WSH004, Oxford Nanopore)</li>
<li>Protocol <code>Flow Cell Wash Kit (EXP-WSH004 or EXP-WSH004-XL)-minion.pdf</code> available on nanopore.com</li>
</ul>
</div>
<div id="steps-4" class="section level3 hasAnchor" number="3.6.2">
<h3><span class="header-section-number">3.6.2</span> Steps<a href="protocol.html#steps-4" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p><strong>Follow the Nanopore protocol. There are three steps:</strong></p>
<ol style="list-style-type: decimal">
<li>Preparation of 400 µl washing buffer</li>
<li>Load 200 µl of washing buffer, wait 5 minutes, then add the remaining 200 µl and wait 1 hour.</li>
<li>If you plan to storage flowcell for later use - add storage buffer. Otherwise load new library.</li>
</ol>
</div>
</div>
<div id="basecalling" class="section level2 hasAnchor" number="3.7">
<h2><span class="header-section-number">3.7</span> Basecalling<a href="protocol.html#basecalling" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Time required: ~ several days depending on computational resources.</p>
<p><strong>Details to follow</strong></p>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent" entry-spacing="0">
<div id="ref-Brewer2020" class="csl-entry">
Brewer, Tess E., Mads Albertsen, Arwyn Edwards, Rasmus H. Kirkegaard, Eduardo P. C. Rocha, and Noah Fierer. 2020. <span>“<span class="nocase">Unlinked rRNA genes are widespread among bacteria and archaea</span>.”</span> <em>ISME Journal</em> 14 (2): 597–608. <a href="https://doi.org/10.1038/s41396-019-0552-3">https://doi.org/10.1038/s41396-019-0552-3</a>.
</div>
<div id="ref-Cusco2018" class="csl-entry">
Cuscó, Anna, Carlotta Catozzi, Joaquim Viñes, Armand Sanchez, and Olga Francino. 2019. <span>“<span class="nocase">Microbiota profiling with long amplicons using Nanopore sequencing: Full-length 16S rRNA gene and the 16S-ITS-23S of the rrn operon</span>.”</span> <em>F1000Research</em> 7. <a href="https://doi.org/10.12688/f1000research.16817.2">https://doi.org/10.12688/f1000research.16817.2</a>.
</div>
<div id="ref-Martijn2019" class="csl-entry">
Martijn, Joran, Anders E. Lind, Max E. Schön, Ian Spiertz, Lina Juzokaite, Ignas Bunikis, Olga V. Pettersson, and Thijs J. G. Ettema. 2019. <span>“<span class="nocase">Confident phylogenetic identification of uncultured prokaryotes through long read amplicon sequencing of the 16S-ITS-23S rRNA operon</span>.”</span> <em>Environmental Microbiology</em> 21 (7): 2485–98. <a href="https://doi.org/10.1111/1462-2920.14636">https://doi.org/10.1111/1462-2920.14636</a>.
</div>
<div id="ref-Srinivas2024" class="csl-entry">
Srinivas, Meghana, Calum J Walsh, Fiona Crispie, Orla O’Sullivan, Paul D Cotter, Douwe Sinderen, and John G Kenny. 2024. <span>“<span class="nocase">Evaluating the Efficiency of 16S-ITS-23S operon Sequencing: A Comparison of Primer Pairs, Sequencing Platforms, and Taxonomic Classifiers</span>.”</span>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="prerequisites.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="references.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/USERNAME/REPO/edit/BRANCH/03-protocol.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["ABlab-workflows-longreads.pdf", "ABlab-workflows-longreads.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
